src/Swagger/Serializer/ApiGatewayNormalizer.php

```diff
@@ -0,0 +1,107 @@
+<?php
+
+/*
+ * This file is part of the API Platform project.
+ *
+ * (c) Kévin Dunglas <dunglas@gmail.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace ApiPlatform\Core\Swagger\Serializer;
+
+use Symfony\Component\Serializer\Normalizer\NormalizerInterface;
+
+/**
```

Comment:
Can you mark this class as `@internal`, so we'll be able to remove it when Amazon will improve its Docker support.

Reply:
Done.